tests/unittests/OCLTest.cpp

```diff
@@ -103,6 +103,37 @@ TEST(OpenCLCorrectnessTest, softmaxGradTest) {
   EXPECT_TRUE(out1.isEqual(out2));
 }
 
+TEST(OpenCLCorrectnessTest, convGradTest) {
+  Tensor inputs(ElemKind::FloatTy, {9, 8, 9, 4});
+  Tensor kernel1(ElemKind::FloatTy, {3, 3, 3, 4});
+  Tensor bias1(ElemKind::FloatTy, {3});
+  Tensor kernel2(ElemKind::FloatTy, {2, 2, 2, 1});
+  Tensor bias2(ElemKind::FloatTy, {2});
+  Tensor selected(ElemKind::IndexTy, {9, 1});
+  inputs.getHandle().initXavier(1);
```

Comment:
nit: I think typically xavier initialization is applied to the weights/bias or other learning params.

Reply:
The test is copied 1:1 from JITTest::convGradTest written by @hegemanjwh2.

@hegemanjwh2 Do you remember what was the intent of this Xavier initialization?